File: CSharp/k06_2.cs
using System;

namespace CSharp
{
    public class K06_2 : IExecutable
    {
        public void Execute()
        {
            for (int i = 0; i < 9; i++)
            {
                for (int j = 0; j < i + 1; j++)
                {
                    Console.Write("*");
                }
                Console.WriteLine();
            }
            Console.WriteLine();

            for (int i = 9; i > 0; i--)
            {
                for (int j = 0; j < i; j++)
                {
                    Console.Write("*");
                }
                Console.WriteLine();
            }
            Console.WriteLine();

            for (int i = 9; i > 0; i--)
            {
                for (int j = 0; j < i - 1; j++)
                {
                    Console.Write(" ");
                }
                
                for (int k = 9; k > i - 1; k--)
                {
                    Console.Write("*");
                }
                Console.WriteLine();
            }
            Console.WriteLine();
        }
    }
}

File: Python/k06_2.py
class k06_2(object):
    
    def execute(self):
        for i in range(0, 9):
            for j in range(0, i + 1):
                print('*', end='')
            print('')
        print('')

        for i in range(9, 0, -1):
            for j in range(0, i):
                print('*', end='')
            print('')
        print('')

        for i in range(9, 0, -1):
            for j in range(0, i - 1):
                print(' ', end='')

            for k in range(9, i - 1, -1):
                print('*', end='')
            print('')
        print('')